CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.9.0-rc19 [unreleased]
+
+### Bugfixes
+- [#2145](https://github.com/influxdb/influxdb/pull/2145): Encode toml durations correctly.
```

Comment:
Let's make this something like "...which fixes default configuration generation."